packages/devtools_app/lib/src/screens/inspector/layout_explorer/ui/utils.dart

```diff
@@ -125,25 +124,24 @@ class Truncateable extends StatelessWidget {
 /// [textColor] color for title text
 class WidgetVisualizer extends StatelessWidget {
   const WidgetVisualizer({
-    Key key,
-    @required this.title,
+    Key? key,
+    required this.title,
     this.hint,
-    @required this.isSelected,
-    @required this.layoutProperties,
+    required this.isSelected,
+    required this.layoutProperties,
     this.child,
     this.overflowSide,
     this.largeTitle = false,
-  })  : assert(title != null),
-        super(key: key);
+  }) : super(key: key);
 
-  final LayoutProperties layoutProperties;
+  final LayoutProperties? layoutProperties;
```

Comment:
non-nullable. I don't see anywhere that null is being passed in for layoutProperties.

Reply:
done